app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.tsx

```diff
@@ -162,43 +162,56 @@ const TableVisualizationRow: React.FC<{
   displayType,
   branchComparison,
 }) => {
+  let field, className;
   const { branch_data, is_control } = results;
-
   const metricData = branch_data[metricKey];
-  const percent = branch_data[METRIC.USER_COUNT]["percent"];
-  const userCountMetric =
-    branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE]["point"];
-
-  const branchType = is_control ? VARIANT_TYPE.CONTROL : VARIANT_TYPE.VARIANT;
-  branchComparison =
-    branchComparison || dataTypeMapping[tableLabel][branchType];
-  const { lower, upper, point, count } = metricData[branchComparison];
-  const significance = metricData["significance"];
-
-  let field;
-  switch (displayType) {
-    case DISPLAY_TYPE.POPULATION:
-      field = populationField(point, percent);
-      break;
-    case DISPLAY_TYPE.COUNT:
-      field = countField(lower, upper, significance, metricName, tableLabel);
-      break;
-    case DISPLAY_TYPE.PERCENT:
-    case DISPLAY_TYPE.CONVERSION_RATE:
-      field = percentField(lower, upper, significance, metricName, tableLabel);
-      break;
-    case DISPLAY_TYPE.CONVERSION_COUNT:
-      field = conversionCountField(count, userCountMetric);
-      break;
-    case DISPLAY_TYPE.CONVERSION_CHANGE:
-      field = conversionChangeField(lower, upper, significance);
-      break;
+
+  if (!metricData) {
+    field = `${metricName} is not available`;
+    className = "text-danger";
```

Comment:
You could just set these as the defaults like

```js
let field = `${metricName} is not available`;
let className = "text-danger";
if(metricData) {
  ...
}
```

Reply:
I like Jared's suggestion but in `if (metricData)`, set `className` to an empty string. When making any sort of default for a `className` make sure it's a string or we get `undefined` rendered on the element. 

<img width="329" alt="image" src="https://user-images.githubusercontent.com/13018240/101214105-14858180-3641-11eb-951a-229798f2c8e5.png">

Reply:
I think you can also use the `classnames` package we have installed (search for `className={classNames`, I _think_ that takes care of an undefined class name but that might be wrong.

Reply:
![](https://user-images.githubusercontent.com/13018240/101214105-14858180-3641-11eb-951a-229798f2c8e5.png)

```css
        .undefined::before {
            content: "¯\\_(ツ)_/¯";
            padding-right: 1em;
            color: red;
        }
```